internal/docstore/dynamodocstore/dynamo.go

```diff
@@ -118,13 +131,22 @@ func (c *collection) put(ctx context.Context, doc driver.Document, condition *st
 		// It doesn't make sense to generate a random sort key.
 		return fmt.Errorf("missing sort key %q", c.sortKey)
 	}
+
+	if av.M[docstore.RevisionField], err = encodeValue(time.Now().UTC()); err != nil {
```

Comment:
I don't think we should use time alone, because it may differ across machines, and it's possible (though unlikely) that we'd get the same value twice. How about `driver.UniqueString`?

Reply:
Changed to use unique string, and that was failing the replay tests. I was doing this because I saw `firedocstore` uses timestamps. Is it feasible for us to make revision format consistent across drivers?

Reply:
Call MakeUniqueStringDeterministicForTesting.

Reply:
Firestore provides the timestamps natively, at the server.

Reply:
That is [called](https://github.com/google/go-cloud/pull/1673/files#diff-df25a5456a33858bf647bc1fb859938aR57) before running conformance tests.